src/view/cli.js

```diff
@@ -37,17 +34,47 @@ CLIView.propTypes = {
   nav: PropTypes.object.isRequired,
 };
 
-const iStyles = StyleSheet.create({
+//
+// Log Output
+//
+
+const logStyles = StyleSheet.create({
+  content: {
+    flexGrow: 1,
+    backgroundColor: color.cliBackground,
+    paddingTop: 25,
+    paddingBottom: 25,
+    paddingLeft: 50,
+    paddingRight: 50,
+  },
   text: {
     textAlign: 'left',
     fontSize: font.sizeS,
   },
 });
 
-const LogItem = ({ text }) => <Text style={iStyles.text}>{text}</Text>;
+class LogOutput extends Component {
+  constructor(props) {
+    super(props);
+    this._ref = React.createRef();
+  }
+
+  get printLogs() {
+    setTimeout(() => this._ref.current.scrollToEnd(), 50);
```

Comment:
@valentinewallace this should be more stable for future react-native-web upgrade.